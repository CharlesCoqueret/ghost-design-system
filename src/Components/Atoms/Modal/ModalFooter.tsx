import React, { PropsWithChildren, ReactElement } from 'react';

import styles from './ModalFooter.module.scss';

const ModalFooter = (props: PropsWithChildren<unknown>): ReactElement => {
  const { children } = props;

  return <div className={styles.container}>{children}</div>;
};

export default ModalFooter;
