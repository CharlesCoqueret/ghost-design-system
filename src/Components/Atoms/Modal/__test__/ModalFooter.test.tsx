import React from 'react';
import { render } from '@testing-library/react';

import ModalFooter from '../ModalFooter';

describe('ModalFooter Component', () => {
  it('ModalFooter renders', async () => {
    const container = render(
      <ModalFooter>
        <div id='CHILD1' />
      </ModalFooter>,
    );

    expect(container).toMatchSnapshot();
  });
});
