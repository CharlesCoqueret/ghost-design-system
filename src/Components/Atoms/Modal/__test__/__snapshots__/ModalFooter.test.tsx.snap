// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalFooter Component ModalFooter renders 1`] = `
<div>
  <div
    class="container"
  >
    <div
      id="CHILD1"
    />
  </div>
</div>
`;
